gitlab/tests/objects/test_users.py

```diff
@@ -135,3 +201,17 @@ def test_user_activate_deactivate(user, resp_activate):
 
 def test_delete_user_identity(user, resp_delete_user_identity):
     user.identityproviders.delete("test_provider")
+
+
+def test_user_follow_unfollow(user, resp_follow_unfollow):
+    user.follow()
```

Comment:
sadly in the test case only the `dict` is returned, so I cannot really assert the attributes of the `User` object.

Any ideas how to convert this to a `User` in the tests? It does work normally

Reply:
Okay that makes sense. Let's keep the conventions then.